test/image/mocks/pie_title_inhole.json

```diff
@@ -0,0 +1,18 @@
+{
+  "data": [
+    {
+      "values": [955, 405, 360, 310, 295],
+      "labels": ["Mandarin", "Spanish", "English", "Hindi", "Arabic"],
+      "textinfo": "label+percent",
+      "hole": 0.3,
+      "title": "Num. speakers",
```

Comment:
Looks like this one is too small already to trigger scaling down - would you mind making this a series including both large and small titles, and perhaps multiline labels in both cases? (Also multiline for outside labels)

Reply:
Done. I added other mocks using multiline titles and scaling.